image_view/package.xml

```diff
@@ -22,6 +22,8 @@
   <depend>image_transport</depend>
   <depend>message_filters</depend>
   <depend>rclcpp</depend>
+  <depend>rclcpp_components</depend>
+  <depend>rosidl_default_generators</depend>
```

Comment:
`rosidl_default_generators` isn't necessary as this package does not create any of its own messages. Would you mind removing this here and from CMakeLists.txt?

Reply:
I reverted the addition of this line from package.xml and squashed into 8630335 and removed the find_package call in c83b533